Calculator/Model/User.cs

```diff
@@ -0,0 +1,28 @@
+namespace Calculator.Model
+{
+    public class User
```

Comment:
I am not .Net expert, by I though there were thing called properties which we could use for that model?